### PR TITLE
Add jenkins badge from CI job to README

### PR DESCRIPTION
**Pull request by obnoxxx**
Signed-off-by: Michael Adam <obnox@samba.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://ci.centos.org/buildStatus/icon?job=gluster_samba-integration)](https://ci.centos.org/view/Gluster/job/gluster_samba-integration/)
```
Comment (nixpanic) [inline README.md:1]: This shows the status of the last build, which includes builds for PRs. I do not know how to pass the branch=master to the URL to get the image. Maybe something to investigate?

Comment (obnoxxx) [reply README.md:1]: ah... good point!

**Comment by anoopcs9** [comment]
retest this please

**Comment by anoopcs9** [comment]
test this please

**Comment by obnoxxx** [comment]
this is far too old and stale. closing.

**Pull request closed without merging**